apps/getting-started-cloudflare-workers/tsconfig.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
This is really comprehensive and a well documented `tsconfig`. For the purposes of this guide I believe it would be most valuable to remove the options currently commented out.